aweber_api/oauth.py

```diff
@@ -61,7 +66,9 @@ def request(self, method, url, data={}, response='body'):
             raise APIException(
                 '{0}: {1}'.format(error_type, error_msg))
 
-        if response == 'body' and isinstance(content, str):
+        if isinstance(content, six.binary_type):
```

Comment:
Thanks for the PR! Looks good aside from needing to pull in six as a requirement. Can you check this without it?

An example from http://python3porting.com/noconv.html#more-bytes-strings-and-unicode:
```
>>> from __future__ import unicode_literals
>>> import sys
>>> if sys.version_info < (3,):
...     text_type = unicode
...     binary_type = str
... else:
...     text_type = str
...     binary_type = bytes
>>> isinstance('U\xf1ic\xf6de', text_type)
True
```

Reply:
Sure. I'll update this